src/main/java/wellnus/reflection/command/LikeCommand.java

```diff
@@ -114,16 +114,11 @@ public String getCommandDescription() {
      * @throws BadCommandException If an invalid command is given
      */
     @Override
-    public void validateCommand(HashMap<String, String> commandMap) throws BadCommandException, NumberFormatException {
+    public void validateCommand(HashMap<String, String> commandMap) throws BadCommandException {
         if (commandMap.size() != ARGUMENT_PAYLOAD_SIZE) {
             throw new BadCommandException(INVALID_ARGUMENT_MSG);
         } else if (!commandMap.containsKey(COMMAND_KEYWORD)) {
             throw new BadCommandException(INVALID_COMMAND_MSG);
-        } else {
-            int questionIndex = Integer.parseInt(commandMap.get(COMMAND_KEYWORD));
-            if (questionIndex > UPPER_BOUND || questionIndex < LOWER_BOUND) {
-                throw new BadCommandException(WRONG_INDEX_MSG);
-            }
         }
         assert getArguments().containsKey(COMMAND_KEYWORD) : COMMAND_KEYWORD_ASSERTION;
```

Comment:
Hmm this assertion can be deleted, the `else if` block just before this already throws an exception for this issue, and an exception is more appropriate in this case in my opinion(user error here, not developer error I believe)